From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Mon, 10 Jul 2023 22:05:08 +0200
Subject: [PATCH] Material fixes - Remove explicit wall sign max stack sizes

Removes the explicit stack sizes for the `Material` wall sign values.
Wall signs only exist in block form, not in item form, so this value has no meaning and is removed for consistency: now all non-item `Material` values have the same `maxStack` value (which is the default value of 64).

License: GPL-3.0 (https://www.gnu.org/licenses/gpl-3.0.html)
Fiddle - https://fiddlemc.org

diff --git a/src/main/java/org/bukkit/Material.java b/src/main/java/org/bukkit/Material.java
index da6e36a16e609272b60fc41ff69a6fa3c34926c0..5ab16a59c2f447565c4671f46324145fe22dd460 100644
--- a/src/main/java/org/bukkit/Material.java
+++ b/src/main/java/org/bukkit/Material.java
@@ -3035,39 +3035,39 @@ public enum Material implements Keyed, Translatable, net.kyori.adventure.transla
     /**
      * BlockData: {@link WallSign}
      */
-    OAK_WALL_SIGN(12984, 16, WallSign.class),
+    OAK_WALL_SIGN(12984, WallSign.class), // Fiddle - remove explicit wall sign max stack sizes
     /**
      * BlockData: {@link WallSign}
      */
-    SPRUCE_WALL_SIGN(7352, 16, WallSign.class),
+    SPRUCE_WALL_SIGN(7352, WallSign.class), // Fiddle - remove explicit wall sign max stack sizes
     /**
      * BlockData: {@link WallSign}
      */
-    BIRCH_WALL_SIGN(9887, 16, WallSign.class),
+    BIRCH_WALL_SIGN(9887, WallSign.class), // Fiddle - remove explicit wall sign max stack sizes
     /**
      * BlockData: {@link WallSign}
      */
-    ACACIA_WALL_SIGN(20316, 16, WallSign.class),
+    ACACIA_WALL_SIGN(20316, WallSign.class), // Fiddle - remove explicit wall sign max stack sizes
     /**
      * BlockData: {@link WallSign}
      */
-    CHERRY_WALL_SIGN(20188, 16, WallSign.class),
+    CHERRY_WALL_SIGN(20188, WallSign.class), // Fiddle - remove explicit wall sign max stack sizes
     /**
      * BlockData: {@link WallSign}
      */
-    JUNGLE_WALL_SIGN(29629, 16, WallSign.class),
+    JUNGLE_WALL_SIGN(29629, WallSign.class), // Fiddle - remove explicit wall sign max stack sizes
     /**
      * BlockData: {@link WallSign}
      */
-    DARK_OAK_WALL_SIGN(9508, 16, WallSign.class),
+    DARK_OAK_WALL_SIGN(9508, WallSign.class), // Fiddle - remove explicit wall sign max stack sizes
     /**
      * BlockData: {@link WallSign}
      */
-    MANGROVE_WALL_SIGN(27203, 16, WallSign.class),
+    MANGROVE_WALL_SIGN(27203, WallSign.class), // Fiddle - remove explicit wall sign max stack sizes
     /**
      * BlockData: {@link WallSign}
      */
-    BAMBOO_WALL_SIGN(18857, 16, WallSign.class),
+    BAMBOO_WALL_SIGN(18857, WallSign.class), // Fiddle - remove explicit wall sign max stack sizes
     /**
      * BlockData: {@link WallHangingSign}
      */
@@ -3360,11 +3360,11 @@ public enum Material implements Keyed, Translatable, net.kyori.adventure.transla
     /**
      * BlockData: {@link WallSign}
      */
-    CRIMSON_WALL_SIGN(19242, 16, WallSign.class),
+    CRIMSON_WALL_SIGN(19242, WallSign.class), // Fiddle - remove explicit wall sign max stack sizes
     /**
      * BlockData: {@link WallSign}
      */
-    WARPED_WALL_SIGN(13534, 16, WallSign.class),
+    WARPED_WALL_SIGN(13534, WallSign.class), // Fiddle - remove explicit wall sign max stack sizes
     POTTED_CRIMSON_FUNGUS(5548),
     POTTED_WARPED_FUNGUS(30800),
     POTTED_CRIMSON_ROOTS(13852),
